src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,9 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+export const UserInfo = ({ user }) => (
+  user.name
+  && user.email
+  && (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
+  )
```

Comment:
try to add more meaningful error handling:
```suggestion
  user.name && user.email ? (
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
    </a>
  ) : 'unknown user'
```